keylime/migrations/versions/eb869a77abd1_create_allowlist_table.py

```diff
@@ -0,0 +1,50 @@
+"""create_allowlist_table
+
+Revision ID: eb869a77abd1
+Revises: 8da20383f6e1
+Create Date: 2021-01-12 10:54:45.263268
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'eb869a77abd1'
+down_revision = 'a09a40352c32'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_registrar():
+    pass
+
+
+def downgrade_registrar():
+    pass
+
+
+def upgrade_cloud_verifier():
+    op.create_table('allowlists',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('name', sa.String(255), nullable=False),
+                    sa.Column('tpm_policy', sa.Text(), nullable=True),
+                    sa.Column('vtpm_policy', sa.Text(), nullable=True),
+                    sa.Column('allowlist', sa.Text(length=429400000), nullable=True),
```

Comment:
I am wondering if we rename this field to ima_policy? It might be confusing to have an allowlist field in an allowlists table :)

Reply:
I'm fine with it as allowlist. I see this field as the main content of the allowlist and the others as meta-data. In that kind of model, double naming is ok.

Reply:
I am asking because ``allowlist`` actually represents IMA in our current implementation, ``ima_policy`` woud match that usage.
There could be other types of allowlist (Pre-Boot ML for example?) that we could not expect at this moment, picking suitable word may help avoid confusion in the future. Ideas?

Reply:
I vote to change the name, precisely to accommodate the case where we have an "allowlist" that would refer to the TPM2 Pre-Boot Event log. Let's make each column clearly refer to the mechanism (IMA or PBEL)

Reply:
After thinking about it, I agree with both you now. Let's rename to ima_policy

Reply:
Will do, I am unable to reach GitHub for a while, hope things will get stabilized soon.